VTEX - Orders API (PII version).json

```diff
@@ -3569,49 +3570,156 @@
+                                        "example": true
+                                    },
+                                    "filterError": {
+                                        "type": "string",
+                                        "description": "Filters orders by error in the workflow processing. You can choose to filter by temporary error (`temp`), permanet error (`perm`) or include all errors (`all`).",
+                                        "example": "all",
+                                        "enum": [
+                                            "all",
+                                            "temp",
+                                            "perm"
+                                        ]
                                     },
                                     "per_page": {
                                         "type": "integer",
-                                        "description": "Number of orders per page.",
-                                        "example": 15
+                                        "description": "Number of orders per page, up to the limit of 100 results.",
+                                        "example": 30
                                     },
                                     "page": {
                                         "type": "integer",
                                         "description": "Number of the page to be retrieved.",
                                         "example": 1
                                     },
+                                    "orderBy": {
+                                        "type": "string",
+                                        "description": "Filters and sorts orders by combining an **OrderField** with a **SortType** value. To do so, you have to concatenate them, as in `OrderField,SortType`. The values accepted are the following:\r\n- **OrderField:** `creationDate`, `orderId`, `totalValue` and `origin`.\r\n- **SortType:** `asc` and `desc`.\r\n\r\nSee below some examples of the `orderBy` field with different `OrderField` values for the `SortType` value `asc`:\r\n- **orderBy creation date:** `creationDate,asc`\r\n- **orderBy order ID:** `orderId,asc`\r\n- **orderBy total value:** `totalValue,asc`\r\n- **orderBy origin:** `origin,asc`",
+                                        "example": "creationDate,asc"
+                                    },
                                     "q": {
                                         "type": "string",
-                                        "description": "Full-text search for the orders.",
-                                        "example": "Postman Test"
+                                        "description": "Full-text search for the orders considering the following fields:\r\n- `orderId`\r\n- `clientEmail`\r\n- `clientDocument`\r\n- `clientName`",
+                                        "example": "customer@email.com"
```

Comment:
are these filters really possible even on a PII endpoint?

Reply:
(feel free to skip this comment if you have already checked with the orders team)

Reply:
Besides, it filters orders, some information might still be masked in the response.